<launch>

<arg name="pkg_name" value="virtual_mirror_qlai"/>
<arg name="veh"  doc="Name of vehicle. ex: megaman"/>
<arg name="local" default="true" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>
<arg name="config" default="baseline" doc="Specify a config."/>
<arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />


<!-- <arg name="testing" default="false" doc="using for test purposes." /> -->

<arg name="node_name" default="virtual_mirror_node"/>


<remap from="virtual_mirror_node/rgb_in" to="/$(arg veh)/camera_node/image/compressed"/>
<remap from="virtual_mirror_node/rgb_out" to="virtual_mirror_node/topic"/>

<node ns="$(arg veh)" pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen">
</node>



<!-- Publication -->
<!-- "~image/raw": sensor_msgs/Image. Raw image by decoding a compressed image in jpeg format.-->
    
<!-- Subscription -->
<!-- "~compressed_image": sensor_msgs/CompressedImage. Input compressed image in jpeg format.-->

</launch>
